src/studio/src/designer/backend/Services/Implementation/TextsService.cs

```diff
@@ -49,5 +62,46 @@ public void DeleteTexts(string org, string repo, string developer, string langua
 
             altinnAppGitRepository.DeleteTexts(languageCode);
         }
+
+        /// <summary>
+        /// Helper method for extracting the markdown filenames from values in a texts objects.
+        /// </summary>
+        /// <param name="texts">Json object consisting of texts with key:value pairs id:text</param>
+        /// <returns>List of markdown filenames</returns>
+        private static List<string> ExtractMarkdownFileNames(Dictionary<string, string> texts)
+        {
+            List<string> markdownFileNames = new List<string>();
+            foreach (KeyValuePair<string, string> text in texts.Where(text => text.Value.StartsWith("${{") && text.Value.EndsWith(".md}}")))
+            {
+                string fileName = text.Value.Substring(3, text.Value.Length - 5);
+                markdownFileNames.Add(fileName);
+            }
+
+            return markdownFileNames;
+        }
+
+        /// <summary>
+        /// Replacing inline markdown from text-values in texts object
+        /// with a reference to a .md file and stores content in that file.
+        /// </summary>
+        /// <param name="languageCode">Language identifier</param>
+        /// <param name="texts">Texts object with inline markdown</param>
+        /// <returns>Tuple of dictionary with keys and texts that
+        /// should be stored as markdown and the original texts objects
+        /// where inline markdown is replaced with the filename.</returns>
+        /// <remarks>Autosave in FE results in old md files that never
+        /// will be overwritten when client change ID.</remarks>
+        private static (Dictionary<string, string> TtextsWithMd, Dictionary<string, string> Ttexts) ExtractMarkdown(string languageCode, Dictionary<string, string> texts)
```

Comment:
Typo TtextsWithMd

Reply:
Think it suppose to be that way since its a Tupple 🤔